src/platform/silabs/wifi/dhcp_client.cpp

```diff
@@ -21,6 +21,14 @@
 #include <stdlib.h>
 #include <string.h>
 
+#if !(SLI_SI91X_MCU_INTERFACE)
+#include "em_bus.h"
```

Comment:
Are these include needed for NCP devices?

Reply:
These headers were included for efr platform and not been used so deleted.